descriptors/ModuleDescriptor-template.json

```diff
@@ -722,6 +722,16 @@
             "finance-storage.transactions.item.put",
             "finance-storage.order-transaction-summaries.item.put"
           ]
+        },
+        {
+          "method": ["POST"],
+          "pathPattern": "/finance/unrelease-encumbrance/{id}",
+          "permissionsRequired": ["finance.unrelease-encumbrance.item.post"],
```

Comment:
please add this permission to finance.all as we did for finance.release-encumbrance.item.post

Reply:
Added